ui/src/views/Home.vue

```diff
@@ -19,42 +19,52 @@
 import axios from "axios";
 
 import Alert from "@/components/Layout/Alert.vue";
+import { store } from "@/store.js";
 
 export default {
   components: {
     Alert
   },
   created() {
     this.config = require("@/config/config.json");
+    if(store.state.isAllowToAccessTheNextPage)
```

Comment:
Is there a way to have the router manage all this? Checking this on every page seems wasteful.